client/landing/stepper/declarative-flow/registered-flows.ts

```diff
@@ -40,6 +40,11 @@ const availableFlows: Record< string, () => Promise< { default: Flow } > > = {
 			/* webpackChunkName: "tailored-ecommerce-flow" */ '../declarative-flow/tailored-ecommerce-flow'
 		),
 
+	wooExpressTrial: () =>
```

Comment:
We might simply want this to be `wooExpress` or `wooexpress` - the trial is not really something we need to call out in the URL.